LayoutTests/imported/w3c/web-platform-tests/trusted-types/trusted-types-svg-script-set-href.html

```diff
@@ -71,7 +71,20 @@
     // but now expect all of them to pass.
     promise_test(t => {
       trustedTypes.createPolicy("default", {
-        createScript: x => x, createHTML: x => x, createScriptURL: x => x });
+        createScript: (x, _, sink) => {
+          //TODO: check sink value.
+          assert_equals(sink, 'HTMLScriptElement src');
```

Comment:
My feeling is that this should be `"SVGScriptElement text"`. But I think it depends what actually triggers this. Maybe run the test in Chromium and see what it actually gets

Reply:
Chromium gives "Element setAttribute".

Reply:
As discussed, I'll move this test to a different patch together with other relevant scriptElement tests.